recipes/recipes_emscripten/llvm/build.sh

```diff
@@ -1,8 +1,26 @@
+(
+  env -i HOME="$HOME" PATH="$PATH" \
+  mkdir native-build
+  cd native-build
+  unset CMAKE_TOOLCHAIN_FILE CMAKE_CROSSCOMPILING_EMULATOR CMAKE_ARGS
+  cmake -G "Unix Makefiles" -S ../llvm -B . \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DLLVM_ENABLE_PROJECTS=clang \
+    -DLLVM_TARGETS_TO_BUILD=host \
+    -DCMAKE_TOOLCHAIN_FILE="" \
+    -DCMAKE_CROSSCOMPILING_EMULATOR=""
+  make llvm-tblgen clang-tblgen -j$(nproc --all)
+  export NATIVE_TOOLS_DIR=$PWD/bin
+  cd ..
+)
+
```

Comment:
Here we want to build llvm-tblgen and clang-tblgen for the host (no cross compilation here)

If emscripten-forge has some configs not allowing this. Probably in the build section we can get llvm-tblgen already built from conda-forge and then just point to it below I geuss !